server/etcdserver/api/membership/confstate_test.go

```diff
@@ -20,8 +20,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"go.etcd.io/etcd/raft/v3/raftpb"
 	"go.etcd.io/etcd/server/v3/etcdserver/api/membership"
-	"go.etcd.io/etcd/server/v3/etcdserver/cindex"
 	betesting "go.etcd.io/etcd/server/v3/mvcc/backend/testing"
+	"go.etcd.io/etcd/server/v3/mvcc/buckets"
```

Comment:
As follow up we should probably rename this to 'schema' eventually (post your refactorings).

Reply:
sg